### PR TITLE
Card

### PR DESCRIPTION
**Pull request by aman11s**
- Added card components
i. Text only card
ii. Card with dismiss
iii. Vertical card
iv. Horizontal card
v. Card with badge
vi. Card with text overlay
vii. Card with shadow

- Netlify preview link is available below.

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    color: #e8415f;
+}
+
+.item-price {
+    font-weight: 700;
+    font-size: 1.2rem;
+}
+
+.card-discription {
+    padding-top: 5px;
```
Comment (SaiArunesh) [inline components/card.css:59]: description

**Review by SaiArunesh — APPROVED** [review]

File: components/card.css (hunk 1 above)
Comment (aman11s) [reply components/card.css:59]: Thanks for reviewing. I had fixed this typo.

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+.overlay-container {
+    opacity: 0.3;
+}
+
+.overlay-text {
+    position: absolute;
+    top: 15rem;
+    right: 2rem;
+    background-color: #f9f9f9;
```
Comment (NamrataRaikwar2002) [inline components/card.css:115]: you can use variables for colors.

**Review by NamrataRaikwar2002 — APPROVED** [review]
all looks fine. only one suggestion.

File: components/card.css (hunk 2 above)
Comment (aman11s) [reply components/card.css:115]: I have not used this color more than 2 times. So is it good to change it to a variable?

**Pull request merged**